torch/csrc/jit/codegen/cuda/lower_trivial_reductions.cpp

```diff
@@ -30,6 +30,7 @@ bool traverseToRFactorTensor(TensorView* tv, IterDomain* root_id) {
 
   const auto& inputs = tv->definition()->inputs();
 
+  // Do we need to add trivial reduction support for grouped reductions?
```

Comment:
I think yes.

Reply:
Actually, the logic seems strange to me. We are checking `inputs.size() != 1`, so it should never accept WelfordOp. It seems this logic is broken for WelfordOp.

Reply:
Pushed a commit to fix it